templates/CRM/Financial/Form/FinancialType.tpl

```diff
@@ -24,46 +24,9 @@
  +--------------------------------------------------------------------+
 *}
 {* this template is used for adding/editing/deleting financial type  *}
-<div class="crm-block crm-form-block crm-financial_type-form-block">
-   {if $action eq 8}
-      <div class="messages status">
-          <div class="icon inform-icon"></div>
-          {ts}WARNING: You cannot delete a financial type if it is currently used by any Contributions, Contribution Pages or Membership Types. Consider disabling this option instead.{/ts} {ts}Deleting a financial type cannot be undone.{/ts} {ts}Do you want to continue?{/ts}
-      </div>
-   {else}
-     <div class="crm-submit-buttons">{include file="CRM/common/formButtons.tpl" location="top"}</div>
-     <table class="form-layout-compressed">
-      <tr class="crm-contribution-form-block-name">
-     <td class="label">{$form.name.label}</td>
-    <td class="html-adjust">{$form.name.html}</td>
-       </tr>
-       <tr class="crm-contribution-form-block-description">
-        <td class="label">{$form.description.label}</td>
-    <td class="html-adjust">{$form.description.html}</td>
-       </tr>
-
-       <tr class="crm-contribution-form-block-is_deductible">
-        <td class="label">{$form.is_deductible.label}</td>
-    <td class="html-adjust">{$form.is_deductible.html}<br />
-        <span class="description">{ts}Are contributions of this type tax-deductible?{/ts}</span>
-    </td>
-       </tr>
-       <tr class="crm-contribution-form-block-is_active">
-        <td class="label">{$form.is_active.label}</td>
-    <td class="html-adjust">{$form.is_active.html}</td>
-       </tr>
-      <tr class="crm-contribution-form-block-is_reserved">
-        <td class="label">{$form.is_reserved.label}</td>
-    <td class="html-adjust">{$form.is_reserved.html}</td>
-       </tr>
-
-      </table>
-   {/if}
-   <div class="crm-submit-buttons">{include file="CRM/common/formButtons.tpl" location="botttom"}</div>
-   {if $action eq 2 or $action eq 4 } {* Update or View*}
-    <div class="crm-submit-buttons">
-       <a href="{crmURL p='civicrm/admin/financial/financialType/accounts' q="action=browse&reset=1&aid=$aid"}" class="button"><span>{ts}View or Edit Financial Accounts{/ts}</a></span>
-    </div>
-   {/if}
-</div>
-
+{include file="CRM/Core/Form/EntityForm.tpl"}
+{if $action eq 2 or $action eq 4 } {* Update or View*}
+  <div class="crm-submit-buttons">
+    <a href="{crmURL p='civicrm/admin/financial/financialType/accounts' q="action=browse&reset=1&aid=$aid"}" class="button"><span>{ts}View or Edit Financial Accounts{/ts}</a></span>
+  </div>
+{/if}
```

Comment:
@eileenmcnaughton I need a help!!

How i can place the div section for button just besides the Save and Cancel buttom? Can i use JS to move the link or will i need to CRM/Core/Form/EntityForm.tpl to include sections insertions?

Reply:
This seems ok?

![screenshot 2018-07-24 03 39 32](https://user-images.githubusercontent.com/336308/43087136-3401e1f4-8ef3-11e8-9bb6-fb7270d02536.png)

Reply:
When you open in a new tab the 'View or Edit Financial Accounts' button goes below of Save and Cancel

Reply:
Does overriding ```protected function addFormButtons()``` { help?

Reply:
buttons look ok for me ![screenshot 2018-07-24 03 39 32](https://user-images.githubusercontent.com/336308/43087504-25379e24-8ef4-11e8-9cc8-760f13edb623.png)

Reply:
hmm not sure maybe @colemanw has a good idea

I guess we could have an 'inner tpl' of

```
  {if $action eq 8}
    <div class="messages status no-popup">
      <div class="icon inform-icon"></div>
      {$deleteMessage|escape}
    </div>
  {else}
    <table class="form-layout-compressed">
      {foreach from=$entityFields item=fieldSpec}
        {assign var=fieldName value=$fieldSpec.name}
        <tr class="crm-{$entityInClassFormat}-form-block-{$fieldName}">
          {include file="CRM/Core/Form/Field.tpl"}
        </tr>
      {/foreach}
    </table>
    {include file="CRM/common/customDataBlock.tpl"}
  {/if}
```
that is included from CRM/Core/Form/EntityForm.tpl - so you can just include the inner if you want ... different buttons